### PR TITLE
Buttons input

### PR DESCRIPTION
**Pull request by itsmebipul00**
Designed Form and buttons

### CODE REVIEW

File: Components/Alerts/Alerts.css [hunk 1]
```diff
@@ -20,30 +19,9 @@
...
+  width: 70%;
 }
 
 .alert-information span {
-  padding: 0.5rem;
-}
-
-.alert-information > * {
-  margin: 3rem;
+  padding: 0.125rem;
```
Comment (zoef-yt) [inline Components/Alerts/Alerts.css:26]: Use px for this

File: Components/Buttons/Buttons.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  border-radius: 50%;
+  background: hsl(var(--clr-dark));
+  color: hsl(var(--clr-white));
+}
+
+.btn-link {
+  color: hsl(var(--clr-dark));
+  margin-left: 1rem;
+  text-decoration: underline;
+  /* text-decoration-color: hsl(var(--clr-dark)); */
```
Comment (zoef-yt) [inline Components/Buttons/Buttons.css]: Remove comment code

File: Components/Buttons/Buttons.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  display: none;
+}
+
+.code-buttons {
+  height: 40vh;
+  width: 70%;
+}
+
+.btn-primary-description {
+  padding: 0.2rem 0.6rem;
```
Comment (zoef-yt) [inline Components/Buttons/Buttons.css:102]: Use px

File: Components/Cards/Cards.css [hunk 4]
```diff
@@ -33,7 +32,7 @@
 
 .cards-verticle-autofit,
 .cards-horizontal-autofit {
-  gap: 2rem;
+  gap: 0.5rem;
```
Comment (zoef-yt) [inline Components/Cards/Cards.css:35]: Use px

File: Components/Form/Form.css [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  background: hsl(var(--clr-white) / 0.7);
+  padding: 2rem;
+  width: 60%;
+}
+
+form input {
+  display: block;
+  width: 100%;
+  margin-bottom: 1.25rem;
+  padding: 0.25rem;
```
Comment (zoef-yt) [inline Components/Form/Form.css:11]: Use px

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Comment by archit7448** [comment]
looking good to merge just some commented codes can be remove

**Review by archit7448 — APPROVED** [review]

File: Components/Buttons/Buttons.css (hunk 2 above)
Comment (itsmebipul00) [reply Components/Buttons/Buttons.css]: Done

File: Components/Alerts/Alerts.css (hunk 1 above)
Comment (itsmebipul00) [reply Components/Alerts/Alerts.css:26]: I guess it is fine, depends on every other person

File: Components/Cards/Cards.css [hunk 6]
```diff
@@ -118,7 +116,7 @@ picture,
   margin: 0;
   width: fit-content;
   height: fit-content;
-  font-weight: 900;
+  font-weight: 400;
   border-radius: 50%;
   padding: 0.22rem 0.5rem;
```
Comment (Omkar-Ghate) [inline Components/Cards/Cards.css:121]: use px here instead of rem

**Review by Omkar-Ghate — COMMENTED** [review]
Except for the above suggestions, it looks good to merge.

**Review by zoef-yt — APPROVED** [review]
Good to go

**Comment by itsmebipul00** [comment]
Thanks guys

**Pull request merged**